src/runtime/libuv.cpp

```diff
@@ -5,21 +5,26 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Author: Markus Himmel, Sofia Rodrigues
  */
 #include <pthread.h>
-#include "runtime/libuv.h"
 #include "runtime/object.h"
+#include "runtime/libuv.h"
 
 namespace lean {
 
 #ifndef LEAN_EMSCRIPTEN
 #include <uv.h>
 
 extern "C" void initialize_libuv() {
+    initialize_libuv_loop();
```

Comment:
I think this made the most sense last, otherwise the initialization and finalization do not happen in consistent orders.

Reply:
If you disagree, please add a comment in the source explaining why the order is inconsistent.

Reply:
I think that I just forgot to change it back.